config/application.rb

```diff
@@ -15,6 +15,10 @@ class Application < Rails::Application
     config.load_defaults 5.2
     config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{rb,yml}')]
 
```

Comment:
I found that this is the API only app, so I have to add these middlewares to support cookies and flash.